src/main/resources/com/atlassian/performance/tools/report/distribution/distribution-comparison-template.html

```diff
@@ -19,7 +19,32 @@
 <canvas id="histogram"></canvas>
```

Comment:
https://travis-ci.com/atlassian/report/builds/104540944 failed
The`DistributionComparisonTest` detected changes in the HTML, so feel free to update the `expected-distribution-comparison.html`

You can use this unit test locally, for a very fast dev loop. In the test stdout you’ll see the link to the freshly generated chart.

Reply:
Done